src/tf/parse_binary_op.cpp

```diff
@@ -0,0 +1,35 @@
+#include <migraphx/tf/op_parser.hpp>
+#include <migraphx/ranges.hpp>
+#include <migraphx/instruction.hpp>
+#include <migraphx/make_op.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace tf {
+
+struct parse_binary_op : op_parser<parse_binary_op>
+{
+    std::vector<op_desc> operators() const
+    {
+        return {{"Add", "add"},
+                {"AddV2", "add"},
+                {"Mul", "mul"},
+                {"Pow", "pow"},
```

Comment:
Maybe we can add the logical ops here as in the onnx side.

Reply:
We have a todo for implementing many operators from onnx into the tf parser.